dht.go

```diff
@@ -5,11 +5,11 @@ import (
 
 	pb "github.com/libp2p/go-libp2p-daemon/pb"
 
-	cid "github.com/ipfs/go-cid"
 	crypto "github.com/libp2p/go-libp2p-crypto"
+	pstore "github.com/libp2p/go-libp2p-peerstore"
 	dht "github.com/libp2p/go-libp2p-kad-dht"
+	cid "github.com/ipfs/go-cid"
```

Comment:
why did this order change? did you run `go fmt` to get those back in order?

Reply:
Not sure why, but I set it back